operator/integration-tests/model-in-image/deploy-operator.sh

```diff
@@ -65,6 +65,7 @@ helm install $OPER_NAME kubernetes/charts/weblogic-operator \
   --set       serviceAccount=$OPER_SA \
   --set       "domainNamespaces={$DOMAIN_NAMESPACE}" \
   --set       "javaLoggingLevel=INFO" \
+  --set       "featureGates=CommonMounts=${DO_CM}" \
```

Comment:
This script is designed so that it can run independently of run-test.sh if desired (where DO_CM is defined).
Instead of expecting DO_CM to be set, it'd probably be fine just to hard code it to true.